ansys/tools/example_coverage/example_coverage.py

```diff
@@ -0,0 +1,144 @@
+import ast
+import os
+import sys
+import argparse
+import pathlib
+
+def find_files(folder_path):
+    """Find all modules available in the folder path provided.
+    The selection is made based on the file's extension.
+    However, __init__.py will be discarded."""
+
+    # Raise an exception if the folder is empty.
+    if len(os.listdir(folder_path)) == 0:
+        raise Exception(f"None file nor folder available in {folder_paths}.")
+
+    module_extension = ('.py')
+    modules = []
+
+    for path, subdirs, files in os.walk(folder_path):
+        for name in files:
+            if name.endswith(module_extension) & (not name.endswith("__init__.py")):
+                modules.append(os.path.join(path,name))
+
+    # Raise an exception if none module was discovered in the input folder.
+    if not modules:
+        raise Exception(f"None module found in: {folder_path}.")
+
+    return modules
+
+def create_report(folder_path):
+    """Write a report to list all modules and the docstring example
+    coverage stats for each of these modules."""
+    print(f'{"Name": <43}{"Methods":>12}{"Missed":>11}{"Covered":>10}')
```

Comment:
@akaszynski I think here "Methods " may not be the most appropriate word. Because it might be functions, classes, properties (which are special kind of methods with decorator). What about "items" or any other suggestions are welcome?

Reply:
Use Docstrings since that's what we're testing for here.